.vscode/launch.json

```diff
@@ -40,5 +50,34 @@
       "internalConsoleOptions": "neverOpen",
       "port": 9230
     },
+    { /* PARTIAL */
+      "name": "[BACKEND] itwin-viewer-desktop",
+      "cwd": "${workspaceFolder}/packages/apps/desktop-viewer-test",
+      "type": "pwa-node",
+      "request": "launch",
+      "console": "internalConsole",
+      "outputCapture": "std",
+      "runtimeExecutable": "${workspaceFolder}/common/temp/node_modules/.bin/electron",
+      "runtimeArgs": [
+        "${workspaceFolder}/packages/apps/desktop-viewer-test/lib/backend/main.js",
+        "--remote-debugging-port=9223"
```

Comment:
itll have the same debugging port as the frontend?

Reply:
this is the remote-debugging-port for the browser, so it needs to match